.github/workflows/qmkpid.yml

```
@@ -0,0 +1,57 @@
+name: Generate QMKPID
+
+on:
+  pull_request:
+    types:
+      - closed
+
+jobs:
+  generate_pid:
+    runs-on: ubuntu-latest
```

I think we should standardise the environment by using our docker container, this should remove the need to install python further down

```suggestion
    runs-on: ubuntu-latest
    
    container: qmkfm/base_container
```

---

Good point 👍